uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/CMSData/UserMailNotifierFactoryTest.php

```diff
@@ -11,17 +11,22 @@
 class CRM_HRCore_CMSData_CMSUserMailNotifierFactoryTest extends CRM_HRCore_Test_BaseHeadlessTest {
 
   public function testItReturnsAnInstanceOfTheExpectedClassWhenCMSIsSupported() {
-    $contactData = ['cmsId' => 1];
-    $mailNotifier = UserMailNotifierFactory::create('Drupal', $contactData);
+    //We need to manually set this value because civicrm sets it to UnitTests
+    //when running in testing environment.
+    CRM_Core_Config::singleton()->userFramework = 'Drupal';
```

Comment:
Don't you need to unset this afterward, to avoid problems with tests that expect this to be "UnitTests"?